src/crypto/tls/handshake_client_tls13.go

```diff
@@ -16,6 +16,46 @@ import (
 	"time"
 )
 
+// EXP_EventTLS13ClientHandshakeTimingInfo carries intra-stack durations for
+// state machine changes. It can be used for tracking metrics during a
+// connection.
+//
+// NOTE: This API is EXPERIMENTAL and subject to change.
```

Comment:
Maybe the note should also include that it should not be included apart of an experiment as it might leak information @cjpatton

Reply:
I'm not sure anything new is exposed here, but it doesn't hurt to be safe. I added a comment to this effect.

Reply:
Thank you!